spanner/integration_test.go

```diff
@@ -454,9 +454,6 @@ loop:
 		if s == nil {
 			break
 		}
-		// This will delete the session on the backend without removing it
-		// from the pool.
-		s.Value.(*session).delete(context.Background())
```

Comment:
This should not be removed. The intention behind this is to verify that the session pool is able to recover from a situation where the backend garbage collects sessions that are still in the pool without the pool knowing about it (or expecting it).

Reply:
Thanks for pointing this out.